src/website/views/components/divider.pug

```diff
@@ -1,4 +1,91 @@
 ---
 title: Divider
-disabled: true
 ---
+
+p.-text Divider is a line separator with some orientation options.
+
+h3 Examples
+p.-text
+| To render a divider, use the class <code>a-divider</code>.
+
+h4 Base
+.example.-mb3
+  .-p3
+    .a-divider
+  :code(lang="html")
+    <div class="a-divider">
+    </div>
+
+h4 Vertical
+.example.-mb3
+  .-p3.-d--flex.-align-items--center
+    | option
+    .a-divider.-vertical
+    | option
+    .a-divider.-vertical
+    | option
+  :code(lang="html")
+    <div>
+      option
+    </div>
+    <div class="a-divider -vertical">
+    </div>
+    <div>
+      option
+    </div>
+    <div class="a-divider -vertical">
+    </div>
+    <div>
+      option
+    </div>
+
+h4 Dark
+.example.-mb3
+  .-p3(style="background-color:#eeeeee")
+    .a-divider.-dark
+  :code(lang="html")
+    <div class="a-divider -dark">
+    </div>
+
+h4 Inverse
+.example.-mb3
+  .-p3(style="background-color:#000000")
+    .a-divider.-inverse
+  :code(lang="html")
+    <div class="a-divider -inverse">
+    </div>
+
+h4 Base with text in a form
+.example.-mb3
+  .-p3
+    .a-divider.-textDivider or
+  :code(lang="html")
+    <div class="a-divider -textDivider">
+      or
+    </div>
+
+h4 Divider Sizes
```

Comment:
This isn't functioning quite right because the border utility is applying border to all sides instead of just top. The result is -b1 is 2px, -b2 is 4px, etc. We need to specify sides in the class names -bt1, -bt2, etc.

Reply:
I  change -b by -bt but i saw the color change in with the modifier -bt because the border modifiers use another border color.   $border-color: rgba(0, 0, 0, 0.12); instead rgba(set-color(black), 0.2);